kiva/trait_defs/ui/wx/kiva_font_editor.py

```diff
@@ -107,20 +108,21 @@ def str_font ( self, font ):
 
         weight    = { kc.BOLD:   ' Bold'   }.get( font.weight, '' )
         style     = { kc.ITALIC: ' Italic' }.get( font.style,  '' )
-        underline = [ '', ' Underline' ][ font.underline != 0 ]
+        underline = ' Underline' if font.underline != 0 else ''
 
         return '%s point %s%s%s%s' % (
                font.size, self.face_name( font ), style, weight, underline )
 
     #---------------------------------------------------------------------------
     #  Returns a list of all available font facenames:
     #---------------------------------------------------------------------------
-
-    def all_facenames ( self ):
+    @staticmethod
+    def all_facenames():
```

Comment:
The switch to staticmethod looks unnecessary, it is called by the parent class on an instance.
The parent class has `all_facenames` as an instance method:
https://github.com/enthought/traitsui/blob/ab90190dded4c70b2ba39e9323b6ef424c926dd4/traitsui/wx/font_editor.py#L122

For consistency, this should probably be kept as an instance method even though its current implementation does not use `self.`

(For forward compatibility, it is probably better to keep it as instance method as it is easier to modify an instance method which has retained access to self than a static method which has almost as much access as a module level function.)

Reply:
I also removed the `@staticmethod` change on `font_face`